src/ui/ui_statusbar.c

```diff
@@ -149,9 +149,30 @@ int statusbar_init(void) {
 void statubar_update(void) {
     char buf[128];
     memset(buf, 0, sizeof(buf));
-    sprintf(buf, "%dS %d.%dV", g_battery.type,
-            g_battery.voltage / 1000,
-            g_battery.voltage % 1000 / 100);
+
+    // display battery voltage
+    int bat_mv = 0;
+    switch (g_setting.power.osd_display_mode)
+    {
```

Comment:
looks like code-style does not align with the clang-format file. if you are using vscode `command palette -> format document` should fix that.